doc/userguide/source-verification.rst

```diff
@@ -0,0 +1,83 @@
+
+Verifying Source Tarball
+========================
+
+Once the Suricata release tarball has been downloaded, the PGP signature should
+be verified. This can be done using the GPG application and is usually
+available on Linux/BSD systems without having to manually install any
+additional packages. For Mac or Windows systems installation packages can be
+found at `<https://gnupg.org/>`_.
+
+Verification Steps
+------------------
+
+These verification steps are for general guidance, the exact process and
+commands may vary between operating systems.
+
+Downloading the sig File
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The sig file needs to be downloaded as well as the tarball. Both files can be
+found at `<https://suricata.io/download/>`_.
```

Comment:
"signature" is how refer to it on the download page, should match I think.

Reply:
will update in next PR